moabb/datasets/Dreyer2023.py

```diff
@@ -0,0 +1,307 @@
+        "SUJ_gender",
+        "Birth_year",
+        "Vision",
+        "Vision_assistance",
+        "Symptoms_TXT",
+        "Level of study",
+        "Level_knowledge neuro",
+        "Meditation practice",
+        "Laterality answered",
+        "Manual activity",
+        "Manual activity TXT",
+    ],
+    "OpenVibe_Perf": ["Perf_RUN_3", "Perf_RUN_4", "Perf_RUN_5", "Perf_RUN_6"],
+    "Mental_Rotation": ["score", "time_1", "time_2"],
+    "PRE_session": [
+        "PRE_Mood",
+        "PRE_Mindfulness",
+        "PRE_Motivation",
+        "PRE_Hours_sleep_last_night",
+        "PRE_Usual_sleep",
+        "PRE_Level_of_alertness",
+        "PRE_Stimulant_doses_12h",
+        "PRE_Stimulant_doses_2h",
+        "PRE_Stim_normal",
+        "PRE_Tabacco",
+        "PRE_Tabacco_normal",
+        "PRE_Alcohol",
+        "PRE_Last_meal",
+        "PRE_Last_pills",
+        "PRE_Pills_TXT",
+        "PRE_Nervousness",
+        "PRE_Awakening",
+        "PRE_Concentration",
+    ],
+    "POST_session": [
+        "POST_Mood",
+        "POST_Mindfulness",
+        "POST_Motivation",
+        "POST_Cognitive load",
+        "POST_Agentivity",
+        "POST_Expectations_filled",
+    ],
+    "Index_of_Learnig_Style": [
+        "active",
+        "reflexive",
+        "sensory",
+        "intuitive",
+        "visual",
+        "verbal",
+        "sequential",
+        "global",
+    ],
+    "16Pf5": [
+        "A",
+        "B",
+        "C_",
+        "E",
+        "F",
+        "G",
+        "H",
+        "I",
+        "L",
+        "M",
+        "N",
+        "O",
+        "Q1",
+        "Q2",
+        "Q3",
+        "Q4",
+        "IM",
+        "EX",
+        "AX",
+        "TM",
+        "IN",
+        "SC",
+        "Interrogation",
+    ],
+}
+
+
+class Dreyer2023(BaseDataset):
+    """Class for Dreyer2023 dataset management. MI dataset.
+
+    .. admonition:: Dataset summary
+
+        ==========  =======  =======  ==========  =================  ============  ===============  ===========
+        Name         #Subj    #Chan    #Classes    #Trials / class    Trials len    Sampling rate    #Sessions
+        ==========  =======  =======  ==========  =================  ============  ===============  ===========
+        Dreyer2023    87       27         2               20              5s            512 Hz           6
+        ==========  =======  =======  ==========  =================  ============  ===============  ===========
+
+    ===================
+    Dataset description
+    ===================
+    A large EEG database with users' profile information for motor imagery
+    Brain-Computer Interface research
+
+    Data collectors : Appriou Aurélien; Caselli Damien; Benaroch Camille;
+                      Yamamoto Sayu Maria; Roc Aline; Lotte Fabien;
+                      Dreyer Pauline; Pillette Léa
+    Data manager : Dreyer Pauline
+    Project leader : Lotte Fabien
+    Project members : Rimbert Sébastien; Monseigne Thibaut
```

Comment:
The dataset summary + description should removed here and put in each dataset (A, B and C) with the correct number of subjects. Keep only the information that this is the parent class and that it should not be instanciated directly.

Reply:
Thanks, @sylvchev ! I moved dataset summary and description to each A, B and C dataset. Since, recording procedure is the same, a large portion of the descriptions is the same for all the three datasets.